src/.r-install.sh

```diff
@@ -175,4 +94,5 @@ cat >>"$1/$2/R/$2" <<EOF
 .onLoad <- function(lib, pkg) .First.lib(lib,pkg)
 .onUnload <- function(lib) .Last.lib(lib)
 EOF
-fi
+
+R CMD INSTALL --no-multiarch --with-keep.source shogun
```

Comment:
Previous method of packaging didn't work with newer versions of R (including the one in CI). In this method I create a package structure
```
- shogun
   - R (contains shogun.R and init.R)
   -  inst (when the package is built, everything inside it is moved there without change)
        - libs (contains shogun.so)   
   - NAMESPACE
   - DESCRIPTION
```
Then use `R CMD INSTALL` to build it to R binary format, and install it.